jbpm-wb-process-instance-migration/pim-react/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-wb-process-instance-migration</artifactId>
+    	<version>7.21.0-SNAPSHOT</version>
+    </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>pim-react</artifactId>
+  <name>Process Instance Migration React GUI</name>
+  <packaging>pom</packaging>
+
+
+
+  <build>
+    <plugins>
+      <plugin>
+	    <groupId>com.github.eirslett</groupId>
+	    <artifactId>frontend-maven-plugin</artifactId>
+	    <version>1.6</version>
+
+        <executions>
+
+          <execution>
+            <id>install node and npm</id>
+            <goals>
+              <goal>install-node-and-npm</goal>
+            </goals>
+            <configuration>
+              <!-- See https://nodejs.org/en/download/ for latest node and npm (lts) versions -->
+              <nodeVersion>v10.13.0</nodeVersion>
+              <npmVersion>6.8.0</npmVersion>
+            </configuration>
+          </execution>
+
+          <execution>
+            <id>npm install</id>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+            <!-- Optional configuration which provides for running any npm command -->
+            <configuration>
+              <arguments>install</arguments>
+            </configuration>
+          </execution>
+
+          <execution>
+            <id>npm run build</id>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+            <configuration>
+              <arguments>run build</arguments>
+            </configuration>
+          </execution>
+
+	  <execution>
+	    <id>run tests</id>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+	    <phase>test</phase>
+            <configuration>
+              <arguments>test</arguments>
+            </configuration>
+	  </execution>
+
+        </executions>
+      </plugin>
+
+	<plugin>
```

Comment:
just wondering .... could this be moved into the parent pom somehow? Ideally in a profile (let's call it e.g. UI), this way user could decide how to build the service - default without UI, or explicitly with ui by invoking the profile during mvn package phase.

Reply:
I would like to keep this in the pim-react instead of parent. Because later we will go back to the split solution, means the pim-react stay in the jbpm-web while the pim-services stays in the droolsjbpm-integration, there won't be a parent.

Reply:
Hm, what I don't like about this is, that pim-react puts sth to pim-service, it should be probably pim-service getting sth from pim-react (rather philosophical question only) ... fair enough. good to stay like it is for now.

Reply:
Jan, I have a discussion with Ruben what's the best approach here to "split" the PIM-GUI and PIM-Services. If it works, we don't need this copy from pim-react to pim-services, or the other way around. (Ruben will carry out the implementation once he is back). What we agree is PIM-GUI will have React and Java part. PIM-GUI (React + java) will be build into a fat jar, and the Java part will have a dependency on PIM-services.